versioned/storage/cassandra/src/main/java/org/projectnessie/versioned/storage/cassandra/CassandraPersist.java

```diff
@@ -539,6 +539,16 @@ public void erase() {
         requests.submitted(backend.executeAsync(ERASE_OBJ, repoId, objId));
       }
     }
+    // We must ensure that the system clock advances a little, so that C*'s next write-timestamp
+    // does not collide with the write-timestamps of the DELETE statements above. Otherwise, the
+    // above DELETEs will silently "overrule" a following INSERT/UPDATE statement. In C*, if a
+    // DELETE and another INSERT/UPDATE have the same write-timestamp, the DELETE wins. This makes
+    // Nessie tests fail on machines that are "fast enough".
+    try {
+      Thread.sleep(2L);
```

Comment:
Would it make sense to spin a bit until the clock advances?... just for the sake of clarity.

Reply:
Don't think so. We can't "sync" to "the database's system clock" anyway.

Reply:
Well, is 2ms enough to compensate for clock differences? 

Anyway, this kind of collision use case is not very probable outside of tests, I think, and as long as 2ms fixes tests, I'm ok :)

Reply:
Yea - this is mostly a test-fix (and 2ms shouldn't harm any real use case).